<launch>
    <!--<rosparam command="load" file="$(find cr17)/param/topics.yaml" />
    <rosparam command="load" file="$(find cr17)/param/misc.yaml" />
-->
    <rosparam command="load" file="$(find cr17)/param/joy.yaml" />

    <node name="high_level_state_controller" type="high_level_state_controller.py" pkg="cr17"/>

    <node name="path_pub" type="path_pub.py" pkg="cr17"/>
</launch>
